ui/app/AppLayouts/Wallet/services/dapps/DAppsRequestHandler.qml

```diff
@@ -124,6 +126,7 @@ SQUtils.QObject {
                 console.error("Error finding network for event", JSON.stringify(event))
                 return null
             }
+            let mainNet = lookupMainnetNetworkFromEvent()
```

Comment:
Could `mainNet` be moved down closer to its usage, e.g just above `let st = getEstimatedFeesStatus`?